FTBFS with Qt 6.10.1

With Qt 6.10.1:
```
/build/gammaray/src/GammaRay/plugins/statemachineviewer/qscxmlstatemachinedebuginterface.cpp: In member function ‘virtual QString GammaRay::QScxmlStateMachineDebugInterface::transitionLabel(GammaRay::Transition) const’:
/build/gammaray/src/GammaRay/plugins/statemachineviewer/qscxmlstatemachinedebuginterface.cpp:215:61: error: no matching function for call to ‘QString::arg(GammaRay::Transition&)’
  215 |     return QStringLiteral("%1 (%2)").arg(events.first()).arg(transition);
      |                                                             ^
/build/gammaray/src/GammaRay/plugins/statemachineviewer/qscxmlstatemachinedebuginterface.cpp:215:61: note: there are 4 candidates
In file included from /usr/include/qt6/QtCore/qobject.h:11,
                 from /usr/include/qt6/QtCore/QObject:1,
                 from /build/gammaray/src/GammaRay/plugins/statemachineviewer/statemachinedebuginterface.h:17,
                 from /build/gammaray/src/GammaRay/plugins/statemachineviewer/qscxmlstatemachinedebuginterface.h:17,
                 from /build/gammaray/src/GammaRay/plugins/statemachineviewer/qscxmlstatemachinedebuginterface.cpp:14:
/usr/include/qt6/QtCore/qstring.h:324:27: note: candidate 1: ‘template<class T, typename std::enable_if<conjunction_v<std::disjunction<std::is_integral<_Tp>, std::conjunction<std::is_enum<_Tp>, std::negation<q23::is_scoped_enum<E, is_enum_v<E> > > > >, std::negation<std::disjunction<std::is_same<typename std::remove_cv<typename std::remove_reference<_Tp>::type>::type, _Float16>, std::is_same<typename std::remove_cv<typename std::remove_reference<_Tp>::type>::type, qfloat16>, std::is_floating_point<_Tp> > >, std::negation<std::conjunction<std::negation<QtPrivate::treat_as_integral_arg<typename std::remove_cv< <template-parameter-1-1> >::type> >, std::is_convertible<T, QAnyStringView> > > >, bool>::type <anonymous> > QString QString::arg(T, int, int, QChar) const’
  324 |     [[nodiscard]] QString arg(T a, int fieldWidth = 0, int base = 10,
      |                           ^~~
/usr/include/qt6/QtCore/qstring.h:324:27: note: template argument deduction/substitution failed:
/usr/include/qt6/QtCore/qstring.h:323:53: error: no type named ‘type’ in ‘struct std::enable_if<false, bool>’
  323 |     template <typename T, if_integral_non_char<T> = true>
      |                                                     ^~~~
/usr/include/qt6/QtCore/qstring.h:339:27: note: candidate 2: ‘template<class T, typename std::enable_if<std::disjunction<std::is_same<typename std::remove_cv<typename std::remove_reference<_Tp>::type>::type, _Float16>, std::is_same<typename std::remove_cv<typename std::remove_reference<_Tp>::type>::type, qfloat16>, std::is_floating_point<_Tp> >::value, bool>::type <anonymous> > QString QString::arg(T, int, char, int, QChar) const’
  339 |     [[nodiscard]] QString arg(T a, int fieldWidth = 0, char format = 'g', int precision = -1,
      |                           ^~~
/usr/include/qt6/QtCore/qstring.h:339:27: note: template argument deduction/substitution failed:
/usr/include/qt6/QtCore/qstring.h:338:50: error: no type named ‘type’ in ‘struct std::enable_if<false, bool>’
  338 |     template <typename T, if_floating_point<T> = true>
      |                                                  ^~~~
/usr/include/qt6/QtCore/qstring.h:344:27: note: candidate 3: ‘template<class T, typename std::enable_if<std::conjunction<std::negation<QtPrivate::treat_as_integral_arg<typename std::remove_cv< <template-parameter-1-1> >::type> >, std::is_convertible<T, QAnyStringView> >::value, bool>::type <anonymous> > QString QString::arg(const T&, int, QChar) const’
  344 |     [[nodiscard]] QString arg(const T &a, int fieldWidth = 0, QChar fillChar = u' ') const
      |                           ^~~
/usr/include/qt6/QtCore/qstring.h:344:27: note: template argument deduction/substitution failed:
/usr/include/qt6/QtCore/qstring.h:343:47: error: no type named ‘type’ in ‘struct std::enable_if<false, bool>’
  343 |     template <typename T, if_string_like<T> = true>
      |                                               ^~~~
/usr/include/qt6/QtCore/qstring.h:364:5: note: candidate 4: ‘template<class ... Args> typename std::enable_if<((sizeof... (Args) >= 2) && conjunction_v<std::conjunction<std::negation<QtPrivate::treat_as_integral_arg<typename std::remove_cv<Args>::type> >, std::is_convertible<Args, QAnyStringView> >...>), QString>::type QString::arg(Args&& ...) const’
  364 |     arg(Args &&...args) const
      |     ^~~
/usr/include/qt6/QtCore/qstring.h:364:5: note: template argument deduction/substitution failed:
/usr/include/qt6/QtCore/qstring.h: In substitution of ‘template<class ... Args> typename std::enable_if<((sizeof... (Args) >= 2) && conjunction_v<std::conjunction<std::negation<QtPrivate::treat_as_integral_arg<typename std::remove_cv<Args>::type> >, std::is_convertible<Args, QAnyStringView> >...>), QString>::type QString::arg(Args&& ...) const [with Args = {GammaRay::Transition&}]’:
```
Works with 6.10.0

--- a/plugins/statemachineviewer/qscxmlstatemachinedebuginterface.cpp
+++ b/plugins/statemachineviewer/qscxmlstatemachinedebuginterface.cpp
@@ -212,7 +212,7 @@ QString QScxmlStateMachineDebugInterface::transitionLabel(Transition transition)
         return QString();
     }
 
-    return QStringLiteral("%1 (%2)").arg(events.first()).arg(transition);
+    return QStringLiteral("%1 (%2)").arg(events.first()).arg(transition.m_id);
 }
 
 State QScxmlStateMachineDebugInterface::transitionSource(Transition transition) const

